staging/src/k8s.io/client-go/pkg/api/BUILD

```diff
@@ -26,6 +26,7 @@ go_library(
     visibility = [
         "//vendor/k8s.io/client-go/pkg/api:__subpackages__",
         "//vendor/k8s.io/client-go/pkg/apis:__subpackages__",
+        "//vendor/k8s.io/client-go/tools/remotecommand:__subpackages__",
```

Comment:
Ah, i don't want to add more dependency to client-go/pkg/api, as we are trying to get rid of this package. Let me check why remotecommand depends on pkg/api.